lms/djangoapps/grades/api/urls.py

```diff
@@ -11,8 +11,14 @@
     '',
     url(
         r'^v0/course_grade/{course_id}/users/$'.format(
-            course_id=settings.COURSE_ID_PATTERN
+            course_id=settings.COURSE_ID_PATTERN,
         ),
         views.UserGradeView.as_view(), name='user_grade_detail'
     ),
+    url(
+        r'^v0/policy/{course_id}/$'.format(
+            course_id=settings.COURSE_ID_PATTERN,
+        ),
+        views.CourseGradingPolicy.as_view(), name='course_grading_policy'
```

Comment:
How about:
   `/api/grades/v0/courses/{course_id}/policy`
That would be consistent in structure with the existing:
   `/api/grades/v0/course_grade/{course_id}/users`
Although one would claim that the latter should just say `courses` instead of `course_grade` since the API is already for `grades`.